plugins/source/aws/resources/services/iam/user_policies.go

```diff
@@ -1,109 +0,0 @@
-package iam
-
-import (
-	"context"
-	"encoding/json"
-	"net/url"
-
-	"github.com/aws/aws-sdk-go-v2/aws"
-	"github.com/aws/aws-sdk-go-v2/service/iam"
-	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
-	"github.com/cloudquery/cq-provider-sdk/provider/diag"
-	"github.com/cloudquery/cq-provider-sdk/provider/schema"
-)
-
-func IamUserPolicies() *schema.Table {
-	return &schema.Table{
-		Name:          "aws_iam_user_policies",
-		Description:   "Inline policies that are embedded in the specified IAM user",
-		Resolver:      fetchIamUserPolicies,
-		IgnoreError:   client.IgnoreCommonErrors,
-		IgnoreInTests: true,
-		Columns: []schema.Column{
-			{
-				Name:        "user_cq_id",
-				Description: "Unique CloudQuery ID of aws_iam_users table (FK)",
-				Type:        schema.TypeUUID,
-				Resolver:    schema.ParentIdResolver,
-			},
-			{
-				Name:        "account_id",
-				Description: "The AWS Account ID of the resource.",
-				Type:        schema.TypeString,
-				Resolver:    client.ResolveAWSAccount,
-			},
-			{
-				Name:        "user_id",
-				Description: "user ID the policy belongs too.",
-				Type:        schema.TypeString,
-				Resolver:    schema.ParentResourceFieldResolver("id"),
-			},
```

Comment:
These fields have been removed as they are relations and can be grabbed from the parent table